deb/control.template

```diff
@@ -0,0 +1,25 @@
+Package: libtuntap-dev
+Version: %VERSION%
+Architecture: %ARCHITECTURE%
+Section: libdevel
+Priority: optional
+Homepage: https://github.com/LaKabane/libtuntap
+Maintainer: LaKabane <libtuntap@rooms.bouledef.eu>
```

Comment:
This is not an email address but an XMPP chat room.

Reply:
What would you suggest as an e-mail?

Reply:
You can put mine `tleguern+libtuntap@bouledef.eu`.

Reply:
Should I leave the name "LaKabane" as it is or add yours as well?

Reply:
No, this part can be kept.

Reply:
My point was only about https://www.debian.org/doc/debian-policy/ch-controlfields.html#s-f-maintainer: between the angled bracket should be an email address. XMPP addresses are easily confused as one.